### DISCUSSION TITLE
Wykorzystanie miarkownika w piecach wędzarniczych

### DISCUSSION THREAD

**Discussion by krzychoooo** [krzychoooo/Miarkownik#9] [General]
Dyskusja na temat wykorzystania miarkownika w piecach wędzarniczych. Jest to kontynuacja wątku ogólnego _Witamy w dyskusji nad Miarkownikiem ciągu WiFi_ rozpoczętego przez piotrux.

**Comment by krzychoooo** [comment]
W piecach wędzarniczych powietrze może bardzo szybko się ogrzać lub schłodzić więc proces regulacji może być odmienny niż CO.
Na początek można zakomentować linie:
 ```
if [piec2#deltaTemp]<-0.2
    TaskValueSet 2,1,50      //set alpha
  endif
  if [piec2#deltaTemp]>0.2
    TaskValueSet 2,1,5      //set alpha
    TaskValueSet 2,3,-10       //set sumError
  endif
```
Czyli będziemy mieli

```
// if [piec2#deltaTemp]<-0.2
//    TaskValueSet 2,1,50      //set alpha
//  endif
//  if [piec2#deltaTemp]>0.2
 //   TaskValueSet 2,1,5      //set alpha
 //   TaskValueSet 2,3,-10       //set sumError
 // endif
```

I poobserwować piec. Jeśli nie będzie zbytnich przeregulowań temperatury to tak zostawić.

**Reply by AndyBoom** [reply]
Ja rozważam przeniesienie na lato miarkownika do szklarni/foliaka, aby regulować kąt otwarcia wywietrznika w zależności od temperatury panującej wewnątrz ;)

**Reply by krzychoooo** [reply]
Fajny pomysł.
